src/js/components/EditsNav.jsx

```diff
@@ -174,5 +174,8 @@ EditsNav.propTypes = {
   syntacticalValidityEditsExist: PropTypes.bool.isRequired,
   qualityVerified: PropTypes.bool.isRequired,
   macroVerified: PropTypes.bool.isRequired,
-  fetched: PropTypes.bool.isRequired
+  editsFetched: PropTypes.bool.isRequired,
+  syntacticalValidityFetched: PropTypes.bool,
```

Comment:
I don't see
- `syntacticalValidityFetched`,
- `qualityFetched`, or
- `macroFetched`

being used here.

Reply:
Those values are being provided by `SubmissionProgressHOC.jsx` to so I figured it would be better to include them in case we want to use them later and to be clear about what props are available.

Reply:
Makes sense. I was just thinking about it the other way, only showing what we're using. Good with me though.